uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -41,13 +42,16 @@
  */
 public class UiAutomatorUtils {
 
+    private final static String TAG = UiAutomatorUtils.class.getSimpleName();
+
     /**
      * Obtain an instance of the UiObject for a given resource id.
      *
      * @param resourceId the resource id of the element to obtain
      * @return the UiObject associated to the supplied resource id
      */
     public static UiObject obtainUiObjectWithResourceId(@NonNull final String resourceId) {
+        Logger.i(TAG, "Obtain an instance of the UiObject for a given resource id..");
```

Comment:
let's also log the resource id in the log message

Reply:
added